R: 0.047

clustering:
    eps: 0.03
    min_samples: 4
